### DISCUSSION TITLE
XDM deos not work

### DISCUSSION THREAD

**Discussion by smestarz** [subhra74/xdm#1233] [General]
There seems to be change in Youtube, the videos are unable to be downloaded. I dont know how to post this issue, but herer it is

**Comment by garteful** [comment]
Same here, it worked great until 2 or 3 days ago, when I've noticed that the addon doesn't promote to download a yt video.

Both links for:
1. Support: https://subhra74.github.io/xdm/redirect.html?target=support 
2. Reporting an issue: https://subhra74.github.io/xdm/redirect.html?target=issue

Doesn't lead to anywhere.

Such a shame, it's an amazing software.

**Comment by Nasgarath** [comment]
Every downloader seems affected. The only ones that still seem to work on YouTube are Vividl (though occasionally it'll fail) and JDownloader 2 (that one, so far, has worked every single time). You could use one of those as a workaround until XDM can fix things on their end (and hopefully they'll figure it out eventually).

**Reply by garteful** [reply]
Thanks for the information.
I found out that also Stacher.IO can handle downloads from yt and other websites efficient. It doesn't have the browser monitoring, but it's a very powerful tool.

Anyway XDM is the best, and I hope it will get fixed soon. 
Funny thing is that XDM works fine with yt on MS Edge browser, but not on Firefox.

**Reply by Nasgarath** [reply]
Oh I wasn't familiar with Stacher.IO but I see it's a frontend for YT-DLP... so is Vividl! So it makes sense, as anything using that as its engine would work.

I have to admit though, Stacher.IO looks better than Vividl (the latter has a very basic interface).

Curious that XDM works as expected on Edge. Makes me wonder if it does on Chrome as well.

**Reply by garteful** [reply]
I don't know about chrome, because I don't use this browser. 
I just installed the XDM addon on Edge, and I was surprised it worked. (I'm not using Edge in a daily basis, just in rare cases like this). 

I wasn't familiar with Stacher.IO either, and last week a Reddit user recommended me to try it, and it's a very nice tool. (You can see some more details on Stacher.IO page on Reddit).
